docs/source/user/envs.rst

```diff
@@ -234,4 +234,11 @@ Variables used specifically by the DJANGO server.
       - optional
       - Probably required if using another db back-end
 
+Docker exceptions
+-------------------------
+
+In Docker environments, the values of :code:`OCT_DJANGO_PORT` and :code:`OCT_BASE_DIR` are overridden and cannot be customized.
+
+To persist data, bind mount the container path :code:`/plugin_data`. The server listens on port :code:`4010`, which should be mapped to the desired host port.

```

Comment:
```suggestion
To persist data, bind mount the container path :code:`/plugin_data`. The server listens on port :code:`4000`, which should be mapped to the desired host port.

```

Inside docker the server is not exposed directly but through `nginx` with port 4000 https://github.com/Crivella/ocr_translate/blob/master/nginx.default